grails-app/views/project/meriPlan/_extendedBaselineMonitoring.gsp

```diff
@@ -9,10 +9,10 @@
     <tbody data-bind="foreach: rows">
     <tr class="header">
         <th class="code"></th>
-        <th class="outcome required">Outcome statement/s</th>
+        <th class="outcome required">${outcomeStatementHeading ?: 'Outcome Statement/s'}</th>
```

Comment:
Looks like most of the other default headings are sentence case - should probably use sentence case here as well.

Reply:
ready for retesting